src/ucs/vfs/sock/vfs_sock.c

```diff
@@ -25,25 +25,27 @@ typedef struct {
 } UCS_S_PACKED ucs_vfs_msg_t;
 
 
-void ucs_vfs_sock_get_address(struct sockaddr_un *un_addr)
+/* Ideally we should use ucs_fill_filename_template here, but ucs library is
+ * not linked by vfs_sock */
+static void ucs_vfs_sock_template(const char *tmpl, char *buf, size_t max)
 {
-    struct passwd *pw;
-    uid_t euid;
+    if (NULL != strstr(tmpl, "%i")) {
+        snprintf(buf, max, tmpl, (int)geteuid());
+    } else {
+        strncpy(buf, tmpl, max);
+    }
+}
 
+void ucs_vfs_sock_get_address(struct sockaddr_un *un_addr, const char *tmpl)
```

Comment:
tmpl -> template?

Reply:
I put `tmpl` like in `ucs_fill_filename_template`, and I think the reason is not to interfere with C++ template keyword (even if it's C)

Reply:
ah yes makes sense